client/src/pages/Membership/HowToJoin/Step.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {
+  Typography,
+  createStyles,
+  WithStyles,
+  withStyles,
```

Comment:
Started in production and no errors as far as I can tell. Is there any benefit to importing from `@material-ui/core/styles` over `@material-ui/core`?

Reply:
Nope none at all. In cases like this it's actually better to import everything from one module rather than 2.
But in cases where you don't use any components from MUI core, it would be better to use the styles module.

Glad things work in prod!